data/brands/amenity/bank.json

```
@@ -6806,6 +6806,20 @@
         "name": "First National Bank Texas"
       }
     },
+    {
+      "displayName": "First Neighborhood Bank",
+      "id": "firstneighborhoodbank",
+      "locationSet": {
+        "include": ["us-wv.geojson"]
+      },
+      "matchNames": ["First National Bank"],
```

```suggestion
      "matchNames": ["fnb"],
```
@Snowysauce Sorry, I used the old name here as I almost mistook it for the First National Bank (USA).
Is the current signed short name okay as match? It is the same matchname as First National Bank.

---

This `matchName` would have the same issue as the first, in that editors using the NSI data for presets/suggestions wouldn't know which entry to present for a bank in West Virginia with the name "FNB". (In iD, it would cycle through both entries, forcing the use of both `brand:wikidata` and `not:brand:wikidata` to fully resolve all warnings.)

I personally think having a `matchName` with this entry does more harm than good. [Everything in West Virginia tagged with "FNB" or "First National Bank" in either name or operator](https://overpass-turbo.eu/s/1Szy) belongs to a different entity than the subject of this PR.